tests/components/cookidoo/test_config_flow.py

```diff
@@ -180,3 +180,116 @@ async def test_flow_user_init_data_already_configured(
 
     assert result["type"] is FlowResultType.ABORT
     assert result["reason"] == "already_configured"
+
+
+async def test_flow_reauth(
+    hass: HomeAssistant,
+    mock_cookidoo_client: AsyncMock,
+    cookidoo_config_entry: MockConfigEntry,
+) -> None:
+    """Test reauth flow."""
```

Comment:
Can you parametrize the test with email input?
- same email
- new email

Reply:
@epenet like  that?